src/DocumentTemplate/tests/test_DocumentTemplate.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+
+class InstanceDictTests(unittest.TestCase):
+    """Testing .._DocumentTemplate.InstanceDict."""
+
+    def test_getitem(self):
+        """The acquisition chain of the object a got method is bound to ...
+
+        does not contain the InstanceDict instance itself.
+
+        This is a test for the fix of the regression described in
+        https://github.com/zopefoundation/Zope/issues/292
+        """
```

Comment:
Our convention is not to use docstrings for testcases, because some testrunners (at least historically) don't display the failing testcase method name if the docstring is present.  In cases like yours, we've just used a comment (runtime discoverability isn't needed).

Reply:
I'll change it for the sake of consistency but the problems of `zope.testrunner` with docstrings are no longer the case.